templates/galaxy/config/file_sources_conf.yml.j2

```
@@ -266,6 +266,8 @@
   doc: Invenio RDM turn-key research data management repository (Sandbox)
   label: Invenio RDM Demo Repository (Sandbox)
   url: https://inveniordm.web.cern.ch/
+  token: ${user.preferences['invenio|token']}
+  public_name: ${user.preferences['invenio|public_name']}
```

```suggestion
  token: ${user.preferences['invenio_sandbox|token']}
  public_name: ${user.preferences['invenio_sandbox|public_name']}
```

Need to use the ID here